00-Raster-Structure.Rmd

```diff
@@ -64,13 +64,11 @@ preferably RStudio, loaded on your computer.
 ### Download Data
 {% include/dataSubsets/_data_Airborne-Remote-Sensing.html %}
 
-****
+## Things To Know To Complete This Lesson
```

Comment:
@mjones01 will you have a look at this before I accept? Is this format in line with our other grey boxes? @brymz just FYI we spent some time making the grey boxes at the top consistent so i'm hesitant to make this change as small as it is because then we have to change ALL of our other lessons. In the future my plan is to automate this box entirely :)